site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -170,6 +169,59 @@ Nav tabs light only differ visually, with a full width bottom border and a diffe
   </li>
 </ul>
 {{< /example >}}
+
+### Nested tabs
+
+Nav tabs light is nested in a tab for adding a level of depth in information organization.
+
+{{< example >}}
+<div role="tablist" aria-owns="nav-tab1 nav-tab2 nav-tab3 nav-tab4">
+  <ul class="nav nav-tabs" id="nav-tab-with-nested-tabs">
+    <li class="nav-item">
+      <a class="nav-link active" aria-current="page" id="nav-tab1" href="#tab1-content" data-bs-toggle="tab" data-bs-target="#tab1-content" type="button" role="tab" aria-controls="tab1-content" aria-selected="true">Tab 1</a>
```

Comment:
There is an error from `npm run docs` that mentions that those 3 links can't have a `type="button"`